cmd/route-emitter/main_test.go

```diff
@@ -891,7 +891,7 @@ var _ = Describe("Route Emitter", func() {
 								}
 
 								return nil
-							}).Should(Succeed())
```

Comment:
Why did we add the timeout here?

Reply:
This was added to get tests to pass locally on our newly slowed down workstations. When run in isolation it passed fine. when run in bulk, it failed after 1s (default), so I bumped it a bit more and everything was good.